src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
```

Comment:
I don't like too much the idea of having this script blocked in an nested main loop waiting for GNOME Software to install something, I feel like way too many things could go wrong with this approach (e.g. What if the user does not install anything? What if the user minimizes GNOME Software and click on the Chrome icon in the meantime?) and I'd rather keep this "post-inst" logic out of this launcher script, which should be limited to launching either Chrome (in which case it stays on while Chrome is running just to keep the sandbox alive) or GNOME Sofware (in which case it should exit as soon as it hands the control over to GS.

I'm not totally sure about it (@joaquimrocha might now better) but I think this post-inst logic belongs to a plugin inside GNOME Software itself, not here.